### PR TITLE
Rubocop ~> 1.0

### PR DESCRIPTION
**Pull request by jodosha**
Describe the changes and motivations for the pull request.

### Review

- [x] All tests are passing.
- [x] Get approval.

### Pre-merge checklist

- [x] The PR relates to a single subject with a clear title and description in grammatically correct, complete sentences.
- [x] Verify that feature branch is up-to-date with `master` (if not - rebase it).
- [x] Double check the quality of [commit messages](http://chris.beams.io/posts/git-commit/).
- [x] Squash related commits together.

### CODE REVIEW

File: granite.gemspec [hunk 1]
```diff
@@ -30,8 +31,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec-its', '~> 1.2 '
   s.add_development_dependency 'rspec-rails', '~> 3.6'
   s.add_development_dependency 'rspec_junit_formatter', '~> 0.2'
-  s.add_development_dependency 'rubocop', '~> 0.78.0'
-  s.add_development_dependency 'rubocop-rails', '~> 2.4.1'
-  s.add_development_dependency 'rubocop-rspec', '~> 1.37.0'
+  s.add_development_dependency 'rubocop', '~> 1.0'
```
Comment (ojab) [inline granite.gemspec:34]: I guess we should lock to minor version and `NewCops: enable`. Right now we wouldn't get any new cops anyway and would get only bugfixes for the current ones, but I doubt there is something to fix.

Comment (jodosha) [reply granite.gemspec:34]: @ojab I set to minor (instead of patch) to encourage us to keep the pace with Rubocop. OK to enable new cops. 👍

Comment (ojab) [reply granite.gemspec:34]: Then we need `NewCops: enable` (see https://docs.rubocop.org/rubocop/versioning.html#pending-cops), otherwise LGTM.

**Review by ojab — APPROVED** [review]

File: granite.gemspec (hunk 1 above)
Comment (jodosha) [reply granite.gemspec:34]: Enabled with this commit: https://github.com/toptal/granite/pull/67/commits/4f0adc5cfe7fdf7d085476d2a159d6e92917adfc

**Pull request merged**